samples/dashboards/demo/datagrid-sync/demo.js

```diff
@@ -24,9 +21,29 @@ const chartOptions = {
     subtitle: {
         text: 'Drag points to update the data grid'
     },
+    accessibility: {
+        point: {
+            descriptionFormat: 'Vitamin A content in {name}: {y} micrograms.'
+        },
+        description: `The first bar chart presenting the ability to edit the
+        connector values by manually changing the height of the bars in the 
+        series, which is possible with allowConnectorUpdate option set to true.`
+    },
+    tooltip: {
+        stickOnContact: true
+    },
+    xAxis: {
+        accessibility: {
+            description: 'categories of food',
+            rangeDescription: ''
```

Comment:
Do we need it?

Reply:
It seems to me that without this, the description of the axis is quite ambiguous, i.e. just "categories". Do you think this is enough or is it better to leave "categories of food"? @marvin19

Reply:
I do not think we need it in this case because the main point of this chart is showing features